packages/google_maps_flutter/google_maps_flutter/example/test_driver/google_maps_e2e.dart

```diff
@@ -823,4 +823,22 @@ void main() {
     iwVisibleStatus = await controller.isMarkerInfoWindowShown(marker.markerId);
     expect(iwVisibleStatus, false);
   });
+
+  testWidgets("fromAssetImage", (WidgetTester tester) async {
+    final Iterable<Future<bool>> futures =
+        <double>[1, 2, 3].map((double pixelRatio) async {
+      final ImageConfiguration imageConfiguration =
+          ImageConfiguration(devicePixelRatio: pixelRatio);
+      final BitmapDescriptor mip = await BitmapDescriptor.fromAssetImage(
+          imageConfiguration, 'red_square.png');
+      final BitmapDescriptor scaled = await BitmapDescriptor.fromAssetImage(
+          imageConfiguration, 'red_square.png',
+          mipmaps: false);
+      mip.toJson().forEach((dynamic x) => print(x));
+      scaled.toJson().forEach((dynamic x) => print(x));
```

Comment:
@duzenko forgot to ask, what is this test testing for? It seems to print stuff but never actually test anything?

Reply:
I was asked to provide a test but not told what the test should do.
I decided that it should load pictures with mipmaps true or false and compare their sizes, for each given size version.

Reply:
The old path should always return the same picture (depending on device pixelRatio)
The new path should return different images depending on the configuration param.

Reply:
It's incomplete obviously because the test was failing to load anything at all or even run on previous flutter versions.

Reply:
I see, thanks. Do you want to try to make it work? I have resolved all the conflicts in this PR, it should be easier to work with now.